src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -324,6 +324,34 @@ public void cannotCreateHoldRequestWhenItemIsNotCheckedOut()
     assertThat(postResponse, hasStatus(HTTP_VALIDATION_ERROR));
   }
 
+  @Test
+  public void cannotCreateRequestItemAlreadyCheckedOutToRequestor()
+    throws InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    MalformedURLException {
+
+      final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+      final IndividualResource rebecca = usersFixture.rebecca();
+      final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+  
+      UUID itemId = smallAngryPlanet.getId();
+  
+      loansFixture.checkOut(smallAngryPlanet, rebecca);
+  
+    //Check RECALL -- should give the same response when placing other types of request.
+      Response postResponse = requestsClient.attemptCreate(new RequestBuilder()
+        .recall()
+        .withItemId(itemId)
+        .withPickupServicePointId(pickupServicePointId)
+        .withRequesterId(rebecca.getId()));
+
+      assertThat(postResponse, hasStatus(HTTP_VALIDATION_ERROR));
+      assertThat(postResponse.getJson(), hasErrorWith(allOf(
```

Comment:
I think the `userId` and `itemId` are relevant parameters for this validation error

```suggestion
    assertThat(postResponse.getJson(), hasErrorWith(allOf(
      hasMessage("This requester currently has this item on loan."),
      hasUUIDParameter("itemId", smallAngryPlanet.getId()),
      hasUUIDParameter("userId", rebecca.getId()))));
```